.github/workflows/codespell-private.yml

```
@@ -2,7 +2,7 @@
 # For general usage in your repo, see the example in codespell.yml
 # https://github.com/codespell-project/codespell
 # Concurrency cancels an action on a given PR once a new commit is pushed
-name: codespell Private Actions
+name: Test
```

I thought this was named as such so other people grab the other one?
```suggestion
name: Test Codespell
```